paymentsheet/src/main/java/com/stripe/android/link/ui/paymentmenthod/PaymentMethodScreen.kt

```diff
@@ -55,10 +55,13 @@ internal fun PaymentMethodBody(
                 .padding(bottom = 32.dp),
             text = stringResource(R.string.stripe_add_payment_method),
             style = LinkTheme.typography.title,
-            color = LinkTheme.colors.textPrimary,
+            color = LinkTheme.colors.typePrimary,
         )
 
-        StripeThemeForLink {
+        StripeThemeForLink(
+            componentBorder = LinkTheme.colors.surfacePrimary,
+            componentDivider = LinkTheme.colors.surfacePrimary
+        ) {
```

Comment:
This forces separators on filled containers (surface_secondary) to look "transparent" (matching the overall background, surface_primary).